projects/pointnav_baselines/experiments/ithor/pointnav_ithor_base.py

```diff
@@ -19,6 +19,8 @@
 class PointNaviThorBaseConfig(ObjectNavBaseConfig):
     """The base config for all iTHOR PointNav experiments."""
 
+    ADVANCE_SCENE_ROLLOUT_PERIOD = 10 ** 13
```

Comment:
Why not have this be `None`?

Reply:
I was going to do that, but I figured that you might have some reason for keeping it like this